.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         with:
           php-version: ${{ matrix.php-versions }}
           coverage: none
-          tools: composer:v1
+          tools: composer:v2
```

Comment:
Please read https://blog.packagist.com/shutting-down-packagist-org-support-for-composer-1-x/